### ISSUE TITLE
NuGet package for Microsoft.Azure.KeyVault.Cryptography won't install in standard project

### ISSUE THREAD

**Issue by davidbending** [Azure/azure-sdk-for-net#2681]
NuGet Package for `Microsoft.Azure.KeyVault.Cryptography` won't install in a .Net Framework 4.5.2 standard class library because of the following error:

```
install-package : Failed to add reference. The package 'Microsoft.Azure.KeyVault.Cryptography' tried to add a framework reference to 'System.Diagnostics.Tools' which was not found in the GAC. This is possibly a 
bug in the package. Please contact the package owners for assistance.
At line:1 char:1
+ install-package Microsoft.Azure.KeyVault.Cryptography
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
```

**Comment by hovsepm** [comment]
@davidbending we do not support .Net 4.5.2. As you can see [here](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/src/KeyVault/Microsoft.Azure.KeyVault.Cryptography/project.json#L26) we only support net45 and Core CLR Netstandard1.5.

**Comment by pomortaz** [comment]
@davidbending  thanks for reporting the issue. I created a PR with the fix: https://github.com/Azure/azure-sdk-for-net/pull/2687

The dependency is not needed and is causing the GAC issue.

**Comment by pomortaz** [comment]
@davidbending the issue is resolved in package version 2.0.5
